ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureIntegrationTest.java

```diff
@@ -68,7 +66,17 @@
 
   @Before
   public void setup() throws Exception {
-    VerifiableProperties verProps = new VerifiableProperties(System.getProperties());
+    String propFileName = "test.properties";
```

Comment:
Where is this file?

Reply:
It can be anywhere in class path.  Mine is in src/test/resources but can't be checked in since it has credentials.

Reply:
I checked it in as a template.